tests/org/tests_create_group_api.py

```diff
@@ -0,0 +1,98 @@
+from org.models import *
+from org.serializers import *
+from tests.AuthAPITestCase import AuthAPITestCase
+from rest_framework.test import APITestCase
+from rest_framework import status
+
+
+class CreateGroupAPITestCase(AuthAPITestCase):
+    """
+    This class is to test the API [post] /api/org/<org_id>/group/
+    present in the view Org.views.GroupView.post 
+    """
+
+    create_group_api = '/api/org/1/group/'
+    invalid_create_group_api='/api/org/2/group/'
```

Comment:
```suggestion
    invalid_create_group_api='/api/org/12345/group/'
```
Lets bring in the practice of using really large org ids for invalid orgs. So that minor changes like adding another org wont break the test cases.